ui/app/src/modules/Content/Authoring/SingleItemSelector.tsx

```diff
@@ -20,7 +20,7 @@ import IconButton from '@material-ui/core/IconButton';
 import { makeStyles } from '@material-ui/core/styles';
 import clsx from 'clsx';
 import { Variant } from '@material-ui/core/styles/createTypography';
-import { SandboxItem } from '../../../models/Item';
+import { DetailedItem } from '../../../models/Item';
```

Comment:
Please check if dialog can work with a BaseItem so that it could receive Sandbox or Detailed item alike.